docs/reference/create.md

```diff
@@ -51,6 +51,7 @@ customize.
        --engine-registry-mirror [--engine-registry-mirror option --engine-registry-mirror option]           Specify registry mirrors to use [$ENGINE_REGISTRY_MIRROR]
        --engine-label [--engine-label option --engine-label option]                                         Specify labels for the created engine
        --engine-storage-driver                                                                              Specify a storage driver to use with the engine
+       --engine-max-attempts                                                                                Specify the number of attempts allowed on waiting period
```

Comment:
I don't think the flag should have the engine prefix (as it's not related to Docker engine's installation / configuration).